hm-service-account-api-rest-v1.yml

```diff
@@ -1364,12 +1364,16 @@ definitions:
     type: array
     title: FleetVehicleGetResponse
     items:
-      $ref: '#/definitions/FleetVin'
+      $ref: '#/definitions/FleetVehicleGetItemResponse'
     example:
-      - vin: WBADT43452G296403
-        status: pending
-      - vin: WBADT43452G296404
+      - vin: VIN0TESTVIN000000
         status: pending
+        brand: mercedes-benz
+        changelog:
+          - timestamp: "2022-03-22T09:36:51"
+            status: pending
+          - timestamp: "2022-03-22T10:36:51"
+            status: approved
```

Comment:
the example data says that the status is pending but we have a changelog with status approved. could you please update the `status` to `approved` ?